template.distillery/Makefile.mobile

```diff
@@ -142,23 +142,23 @@ endif
 
 $(APPJS): $(JS_PREFIX).js
 ifeq ($(APP_REMOTE),yes)
-	APPJS_FILE=$$(curl -s -f $(APP_SERVER)|grep -E -o -m 1 '$(PROJECT_NAME)_[a-f0-9]*?\.js') &&\
+	APPJS_FILE=$$(curl -s -f $(APP_SERVER) | cat | grep -E -o -m 1 '$(PROJECT_NAME)_[a-f0-9]*?\.js') &&\
```

Comment:
@pwbs thanks, no need to change it. I was curious about what crazy glitch necessitates it :). Maybe add a comment when you get a chance, because I am sure somebody will propose removing the useless `cat`.